gateway/Dockerfile

```diff
@@ -1,3 +1,18 @@
+FROM quay.io/kaine/spmm-pipeline-base:latest AS builder
```

Comment:
For this image:
* Where is its Docker file?
* Can we push it to factory2 instead of your personal registry?

Reply:
> For this image:
> 
> * Where is its Docker file?
> * Can we push it to factory2 instead of your personal registry?

I'll change the base image